scipy/linalg/_matfuncs.py

```diff
@@ -208,25 +211,42 @@ def logm(A, disp=True):
 
 
 def expm(A):
-    """
-    Compute the matrix exponential using Pade approximation.
+    """Compute the matrix exponential of an array.
 
     Parameters
     ----------
-    A : (N, N) array_like or sparse matrix
-        Matrix to be exponentiated.
+    A : ndarray
+        Input with last two dimensions are square ``(..., n, n)``.
 
     Returns
     -------
-    expm : (N, N) ndarray
-        Matrix exponential of `A`.
+    eA : ndarray
+        The resulting matrix exponential with the same shape of A
```

Comment:
Should be double backticks. Single backticks in sphinx mean the default role, which is [configured](https://github.com/scipy/scipy/blob/master/doc/source/conf.py#L99) to be `autolink` in scipy. Inline code is delimited with double backticks.

Reply:
No @adeak is correct with single backticks here. All arguments, methods, class (direct links) are single ticked while more general code is double backticked.

Reply:
Looking at the [autosummary documentation](https://github.com/sphinx-doc/sphinx/blob/e719b8277082c459574b7979009b27d2e90e83ef/sphinx/ext/autosummary/__init__.py#L44-L49), `autolink` on unresolved objects renders as a simple emphasis. Checking the current docs I see that `A` is indeed not resolved, and produces `<p>Matrix logarithm of <em class="xref py py-obj">A</em></p>.` At the same time, *resolved* objects render as preformatted text: `<code class="xref py py-mod docutils literal notranslate"><span class="pre">scipy.integrate</span></code>`

I was unable to find a reference for the current backtick guidelines, however what you decribe possibly leads to worse output readability because emphasis `<em>` is easier to confuse with regular text than code `<pre>`.

I also don't mean to distract or extend this discussion, please feel free to ignore this consideration.

Reply:
Usually the screen estate for this part is very small so it won't make any difference to jump up 2 cm but it seems that `A` is more readable than _A_ in the online documentation so I went ahead with double but point taken about the parameter linking.